topo_processor/data/data_transformers/data_transformer_repo.py

```diff
@@ -26,8 +26,8 @@ def transform_data(self, item: Item) -> None:
                     transformer.transform_data(item)
                 except Exception as e:
                     item.add_error(str(e), transformer.name, e)
-                    get_log().warning(f"Data Transform Failed: {e}", transformers=transformer.name)
-                    return
+                    get_log().error(f"Data Transform Failed: {e}", transformers=transformer.name)
```

Comment:
I wonder if it is the error is not display in a better way (specially in Elastic Search) if you declare it as an object like:
```
get_log().error(f"Data Transform Failed", transformers=transformer.name, error[or err]=e)
```